Update reference documentation (add data model and GeoPackage) (#310)
Open-Telecoms-Data/open-fibre-data-standard: docs/reference/publication_formats/geopackage/index.md

# GeoPackage

> This page describes a standardised representation of the [OFDS data model](../../data_model.md) as a [GeoPackage](https://www.geopackage.org/). A GeoPackage is a [SQLite](https://sqlite.org/) database. This page provides an [overview](#overview) of the structure of an OFDS GeoPackage, and detailed [definitions](#table-definitions) for each of the tables in the database.

Reviewer: ```suggestion
This page describes how to publish [OFDS conformant data](../../data_model.md) as a [GeoPackage](https://www.geopackage.org/). A GeoPackage is a [SQLite](https://sqlite.org/) database. This page provides an [overview](#overview) of the structure of an OFDS GeoPackage, and detailed [definitions](#table-definitions) for each of the tables in the database.
```
Echoing the language I've suggested elsewhere.

Author: I'd prefer to avoid framing a reference page as a how-to. I'm also trying to avoid focussing too much on publication as the only use case for the standard - from what we've learned so far, data storage and exchange might resonate more with the readers.

In-line with my response to your other suggestion, how about:

```suggestion
This page describes the schema for publishing, storing or exchanging data that conforms to the [OFDS data model](../../data_model.md) as a [GeoPackage](https://www.geopackage.org/). A GeoPackage is a [SQLite](https://sqlite.org/) database. This page provides an [overview](#overview) of the structure of an OFDS GeoPackage, and detailed [definitions](#table-definitions) for each of the tables in the database.
```

Edit: Once we've agreed on a wording, we'll need to update the introductory sentences on the reference pages for the other data formats.

Reviewer: Yup - that works

The OFDS GeoPackage format is based on [GeoPackage 1.4.0](https://www.geopackage.org/spec140/), including the [GeoPackage Schema Extension](https://www.geopackage.org/spec140/#extension_schema) and [GeoPackage Related Tables Extension](https://docs.ogc.org/is/18-000/18-000.html).

## Template

The [OFDS GeoPackage template](../../../../schema/geopackage/network-schema.gpkg) implements the structure described on this page.

```{tip}
You can explore the structure of the OFDS GeoPackage template in common GIS tools such as [QGIS](https://qgis.org/), or you can connect directly to the SQLite database using your preferred SQL client. 
```

## Overview

> The following diagram illustrates how the main entities and relationships in the OFDS data model are represented in an OFDS GeoPackage. Features (spatial entities) are coloured yellow, non-spatial entities are coloured blue, and associative tables (M:N relationships) are coloured grey.

Reviewer: Consider adding to the end of this para: 

Only primary and foreign keys are shown in the diagram. See the [Table definitions](#table-definitions) section below for a complete description of columns.

Author: Done in 22c7120cf6b0b1e74acc03ac83fa77fb1537aa24
